recipes/server.rb

```diff
@@ -37,12 +37,14 @@
 
 execute 'update-splunk-mgmt-port' do
   command "#{splunk_cmd} set splunkd-port #{node['splunk']['mgmt_port']} -auth '#{splunk_auth_info}'"
+  sensitive true
```

Comment:
Are these to actually avoid anything sensitive?

Reply:
the splunk admin username and password are part of the command line arguments, so they shouldn't be displayed.

Reply:
oh yep that makes sense then